Author: Roland Rosenfeld <roland@debian.org>
Description: Add ru_RU.KOI8-R.ps for russion KOI8-R output (Closes: #310738).
Bug-Debian: http://bugs.debian.org/310738

--- a/fig2dev/dev/Imakefile
+++ b/fig2dev/dev/Imakefile
@@ -69,5 +69,6 @@
 	  $(LN) cs_CZ.ps hr_HR.ps; $(LN) cs_CZ.ps hu_HU.ps; \
 	  $(LN) cs_CZ.ps ro_RO.ps; $(LN) cs_CZ.ps pl_PL.ps; \
 	  $(LN) cs_CZ.ps sk_SK.ps; $(LN) cs_CZ.ps sl_SI.ps )
+	InstallNonExecFile(ru_RU.KOI8-R.ps,FIG2DEV_LIBDIR)
 #endif
 
